pkg/frontend/transport/handler.go

```diff
@@ -266,7 +275,7 @@ func (f *Handler) reportQueryStats(r *http.Request, queryString url.Values, quer
 	}
 
 	// Log stats.
-	logMessage := append([]interface{}{
+	logMessage := append([]any{
 		"msg", "query stats",
 		"component", "query-frontend",
 		"method", r.Method,
```

Comment:
[nit] WDYT adding `status_code` as a field here as well? It will make filtering by it with LogQL very easy.